teammapper-frontend/mmp/src/map/handlers/draw.ts

```diff
@@ -399,34 +446,46 @@ export default class Draw {
         }
     }
 
+    /**
+     * Check if given node has hidden children and render the eye icon if so
+     * @param {Node} node
+     */
+    private checkForHiddenChildren(node: Node) {
```

Comment:
this method does not only check, but actually hides or even removes Dom icons. That should be reflected in the name, as its otherwise confusing